src/app/admins/admins.pipe.ts

```diff
@@ -2,28 +2,22 @@ import { Pipe, PipeTransform } from "@angular/core";
 import { Store } from "@ngxs/store";
 import { IAdmin } from "./admins.interfaces";
 import { AdminsState } from "./state/admins.state";
+import { AuthService } from "../core/auth/auth.service";
+import { switchMap } from "rxjs/operators";
```

Comment:
@babul3012 could you remove the unused imports from this file such as switchMap etc?